src/tests/calculator.spec.ts

```diff
@@ -11,6 +12,17 @@ test.describe('smoke', () => {
         await calculatorPage.acceptCookiesIfAppear();
     });
 
+    test('should estimate the cost of Compute Engine instance with valid inputs', async ({page}) => {
```

Comment:
Consider using custom fixtures to provide page objects to test
https://playwright.dev/docs/test-fixtures#creating-a-fixture